doc/guides/abstraction/posix.rst

```diff
@@ -0,0 +1,432 @@
+standard requires all basic thread services, except those related to
+multiple processes.
+
+
+.. csv-table:: POSIX_THREADS_BASE
+   :header: API, Status
+   :widths: 50,10
+
+    pthread_atfork(),|crossmark|
+    pthread_attr_destroy(),|checkmark|
+    pthread_attr_getdetachstate(),|checkmark|
+    pthread_attr_getschedparam(),|checkmark|
+    pthread_attr_init(),|checkmark|
+    pthread_attr_setdetachstate(),|checkmark|
+    pthread_attr_setschedparam(),|checkmark|
+    pthread_cancel(),|checkmark|
+    pthread_cleanup_pop(),|crossmark|
+    pthread_cleanup_push(),|crossmark|
+    pthread_cond_broadcast(),|checkmark|
+    pthread_cond_destroy(),|crossmark|
+    pthread_cond_init(),|checkmark|
+    pthread_cond_signal(),|checkmark|
+    pthread_cond_timedwait(),|checkmark|
+    pthread_cond_wait(),|checkmark|
+    pthread_condattr_destroy(),|crossmark|
+    pthread_condattr_init(),|crossmark|
+    pthread_create(),|checkmark|
+    pthread_detach(),|checkmark|
+    pthread_equal(),|crossmark|
+    pthread_exit(),|checkmark|
+    pthread_getspecific(),|checkmark|
+    pthread_join(),|checkmark|
+    pthread_key_create(),|checkmark|
+    pthread_key_delete(),|checkmark|
+    pthread_kill(),|crossmark|
+    pthread_mutex_destroy(),|checkmark|
+    pthread_mutex_init(),|checkmark|
+    pthread_mutex_lock(),|checkmark|
+    pthread_mutex_trylock(),|checkmark|
+    pthread_mutex_unlock(),|checkmark|
+    pthread_mutexattr_destroy(),|crossmark|
+    pthread_mutexattr_init(),|crossmark|
+    pthread_once(),|checkmark|
+    pthread_self(),|checkmark|
+    pthread_setcalcelstate(),|crossmark|
+    pthread_setcanceltype(),|crossmark|
+    pthread_setspecific(),|checkmark|
+    pthread_sigmask(),|crossmark|
+    pthread_testcancel(),|crossmark|
+
+
+
+XSI_THREAD_EXT
+++++++++++++++
+
+The XSI_THREADS_EXT Unit of Functionality is required because it provides
+functions to control a thread's stack. This is considered useful for any
+real-time application.
+
+The following services are supported:
+
+.. csv-table:: XSI_THREAD_EXT
+   :header: API, Status
+   :widths: 50,10
+
+    pthread_attr_getguardsize(),|crossmark|
+    pthread_attr_getstack(),|checkmark|
+    pthread_attr_setguardsize(),|crossmark|
+    pthread_attr_setstack(),|checkmark|
+    pthread_getconcurrency(),|crossmark|
+    pthread_setconcurrency()|crossmark|
+
+
+XSI_THREAD_MUTEX_EXT
+++++++++++++++++++++
+
+The XSI_THREAD_MUTEX_EXT Unit of Functionality is required because it has
+options for controlling the behavior of mutexes under erroneous application use.
+
+
+The following services are supported:
+
+.. csv-table:: XSI_THREAD_MUTEX_EXT
+   :header: API, Status
+   :widths: 50,10
+
+    pthread_mutexattr_gettype(),|checkmark|
+    pthread_mutexattr_settype(),|checkmark|
+
+
+POSIX_C_LANG_SUPPORT
+++++++++++++++++++++
+
+The POSIX_C_LANG_SUPPORT Unit of Functionality contains the general ISO C
+Library.
+
+
+.. csv-table:: POSIX_C_LANG_SUPPOR
+   :header: API, Status
```

Comment:
Would it make sense clarifying that this status is the support in Zephyr's provided minimal libc?, Or maybe in the text above, and/or even in the overall introduction.